kernelstub/installer.py

```diff
@@ -118,10 +118,17 @@ def setup_kernel(self, kernel_opts, setup_loader=False, overwrite=False, simulat
         self.log.debug('kernel being copied to %s' % self.kernel_dest)
 
         try:
-            self.copy_files(
-                self.opsys.kernel_path,
-                self.kernel_dest,
-                simulate=simulate)
+            arch = platform.uname()[4]
```

Comment:
The uname is a named tuple, so it's clearer to access by field name rather than index. But `platform.machine()` apparently also exists to return the same thing.

Reply:
Oh, you're right. Will fix.